### DISCUSSION TITLE
Way to modify color of antd Menu and Search component?

### DISCUSSION THREAD

**Discussion by WilliamUW** [ant-design/ant-design#33939] [Q&A]
![image](https://user-images.githubusercontent.com/25058545/152656836-89650132-b2be-4c26-9ca7-637c086c20df.png)

I currently have an antd Menu and Search Button component with the default blue color. I was wondering if I can modify that color to red for example. 

I tried using color and background color, but those either modified the text color or made the entire block red. I only want to replace the blue color that shows up with red. 

Thank you in advance.

**Comment by zhushenhang** [comment]
mark. i have a same question
